actions/project.search.by.property.yaml

```
@@ -105,6 +105,7 @@ parameters:
       - "is_enabled"
       - "name"
       - "parent_id"
+      - "null"
```

I thing it needs to not be a string. Like in the `image.search.by.expression.yaml`
```suggestion
      - null
```